cmake/modules/FindGLEW.cmake

```diff
@@ -62,6 +62,12 @@ if (WIN32)
         PATH_SUFFIXES
             Release/${ARCH}
         DOC "The GLEW library")
+
+    if ("${GLEW_LIBRARY}" MATCHES "glew32s(d|)")
+        add_definitions("/DGLEW_STATIC")
```

Comment:
````
/*
 * GLEW_STATIC is defined for static library.
 * GLEW_BUILD  is defined for building the DLL library.
 */

#ifdef GLEW_STATIC
#  define GLEWAPI extern
#else
#  ifdef GLEW_BUILD
#    define GLEWAPI extern __declspec(dllexport)
#  else
#    define GLEWAPI extern __declspec(dllimport)
#  endif
#endif
````
If you don't have GLEW_STATIC, and you do have GLEW_BUILD, USD will attempt to re-export the symbols, so if you use GLEW yourself, and link glf, then the linker will report duplicate symbols? I agree that these macros in glew.h are bogus, extern is seldom what you want in this case. 

In Zeno, I worked around this issue by requiring Zeno libs to link the PlatformGL library, and never glew directly. Maybe this hiccough needs to be documented...